src/domUpdates.js

```diff
@@ -63,10 +67,47 @@ let domUpdates = {
   </article>`
     });
   },
+
+
+
+    displayUpcomingRecipes(currentUser, recipeDatas) {
```

Comment:
We did this one together, and it works, but it definitely does not follow SRP.  This one can be refactored tomorrow.